torch/csrc/jit/codegen/cuda/lower_index.cpp

```diff
@@ -221,6 +221,83 @@ void IndexLowering::handle(const IndexSelectOp* sop) {
   GpuLower::current()->propagateExprInfo(sop, back());
 }
 
+namespace {
+Val* packToRealIndices(kir::TensorIndex* idx) {
+  Val* real_indices = nullptr;
+  bool first = true;
+  for(auto x : idx->indices()) {
+    if(first) {
+      real_indices = x;
+      first = false;
+    }
+    else {
+      real_indices = SimplifyingIrBuilder::addExpr(real_indices, x);
+    }
+  }
+  return real_indices;
+}
+
+std::vector<Val*> getStrideForEachAxis(std::vector<IterDomain*> &domain) {
+  std::vector<Val*> strides(1, IrBuilder::create<Int>(1));
+  Val* stride = IrBuilder::create<Int>(1);
```

Comment:
It seems the `stride` object created in 242 line is never pushed into the `strides` container?

Reply:
Yes, repeated here.